roles/azure_virtual_machine_with_public_ip/README.md

```diff
@@ -15,22 +15,22 @@ Role Variables
   - **name**: (Required) Name of the virtual machine.
   - **admin_username**: Administrator's login name of a server. Required for creation.
-  - **admin_password**: Password of the administrator login. Not required when `os='Linux'` and `ssh_pw_enabled='false'`.
+  - **admin_password**: Password of the administrator login. Not required when `os='Linux'` and `ssh_pw_enabled='false'`. The supplied password must be between 6-72 characters long, control characters are not allowed, and must satisfy at least 3 of password complexity requirements from the following: Contains an uppercase character; Contains a lowercase character; Contains a numeric digit; Contains a special character. 
   - **size**: Valid Azure VM size. Choices vary depending on the subscription and location.
   - **network_interfaces**: List of network interfaces to add to the VM. Can be a string of name or resource ID of the network interface, can also be a dict containing 'resource_group' and 'name' of the network interface. A default network interface will be created if not provided.
   - **vnet_address_prefixes_cidr**: List of IPv4 address ranges for virtual network where each is formatted using CIDR notation.
   Required when creating a new virtual network, otherways should be omitted
   - **subnet_address_prefixes_cidr**: CIDR defining the IPv4 and IPv6 address space of the subnet.
   Required when creating a new virtual network, otherways should be omitted
+  - **load_balancer_backend_address_pools**: List of existing load balancer backend address pools in which the network interface will be load balanced.
   - **os**: Type of Operating System. Default is 'Linux'
-  - **availability_set**: Name or ID of existing availability set to add the VM to.
   - **image**: The image used to build the VM. For custom images, the name of the image. To narrow the search to a specific resource group, a dict with the keys name and resource_group. For Marketplace images, a dict with the keys publisher, offer, sku, and version. Set version=latest to get the most recent version of a given image.
-  - **ssh_pw_enabled**: Enable/disable SSH passwords. Valid values are 'yes', 'no'. Default value is 'yes'.
-  - **ssh_public_keys**: List of SSH keys when `os='Linux'`. Accepts a list of dicts where each dictionary contains two keys, 'path' and 'key_data'. Set path to the default location of the authorized_keys files. For example, path=/home/<admin username>/.ssh/authorized_keys. Set key_data to the actual value of the public key.
+  - **ssh_pw_enabled**: Enable/disable SSH passwords. Valid values are 'true', 'false'. Default value is 'true'.
+  - **ssh_public_keys**: List of SSH keys when `os='Linux'` and `ssh_pw_enabled='false'`. Accepts a list of dicts where each dictionary contains two keys, 'path' and 'key_data'. Set path to the default location of the authorized_keys files. For example, path=/home/<admin username>/.ssh/authorized_keys. Set key_data to the actual value of the public key.
```

Comment:
Are you sure you can't specify ssh keys if you also allow password login?

Reply:
good point. I'll rephrase

Reply:
fixed